### ISSUE TITLE
--add-soft-deletes broken with v4.7.1

### ISSUE THREAD

**Issue by caseylucas** [volatiletech/sqlboiler#1009]
After upgrading from version 4.6.0 to 4.7.1, the `--add-soft-deletes` option no longer works. Instead it fails with:
```
Error: unable to generate output: failed to format template

 1182           args = queries.ValuesFromMapping(reflect.Indirect(reflect.ValueOf(o)), datasetPrimaryKeyMapping)
 1183           sql = "DELETE FROM \"concat\".\"dataset\" WHERE \"dataset_id\"=$1"
 1184   } else {
 1185           currTime := time.Now().In(boil.GetLocation())
 1186           o.DeletedAt = null.TimeFrom(currTime)
>>>>            wl := []string{(or $.AutoColumns.Deleted "deleted_at")}
 1188           sql = fmt.Sprintf("UPDATE \"concat\".\"dataset\" SET %s WHERE \"dataset_id\"=$2",
 1189                   strmangle.SetParamNames("\"", "\"", 1, wl),
 1190           )
 1191           valueMapping, err := queries.BindMapping(datasetType, datasetMapping, append(wl, datasetPrimaryKeyColumns...))
 1192           if err != nil {

: 1187:22: illegal character U+0024 '$' (and 10 more errors)
```

The table in question is called "dataset" and does have a "deleted_at" column defined:
```
create table dataset
(
    dataset_id  bigserial     not null primary key,
    dataset_xid varchar(255)  not null unique,
    description varchar(1024) not null,
    created_at  timestamptz   not null default current_timestamp,
    updated_at  timestamptz   not null default current_timestamp,
    deleted_at  timestamptz
);
```

### What version of SQLBoiler are you using (`sqlboiler --version`)?

SQLBoiler v4.7.1

### What is your database and version (eg. Postgresql 10)

Postgres 13.3

### If this happened at generation time what was the full SQLBoiler command you used to generate your models? (if not applicable leave blank)

sqlboiler --add-soft-deletes -o ../internal/boiler/ -c sqlboiler.toml psql

**Comment by caseylucas** [comment]
CC @stephenafamo I assume this issue is based on recent changes you made but am not sure.

**Comment by ping-localhost** [comment]
I am having the same issue using MariaDB ver. 10.6.4.

**Comment by tomsharratt** [comment]
This is fixed in 4.8.1
